public/lesswrong-editor/src/editorConfigs.ts

```diff
@@ -59,10 +59,10 @@ const embedConfig = {
 		},
 		{
 			name: "Neuronpedia",
-			url: /^neuronpedia\.org\/([a-zA-Z0-9-/]*)(?:\?embed=true&embedexplanation=([a-zA-Z0-9-/]*)&embedplots=([a-zA-Z0-9-/]*)&embedtest=([a-zA-Z0-9-/]*)&height=([a-zA-Z0-9-/]*))?/,
-			html: ([match, identifier, explanation, plots, test, height]: RegExpMatchArray) => `
+			url: /^neuronpedia\.org\/([a-zA-Z0-9-/]*)(?:\?embed=true(.*))?/,
```

Comment:
@hijohnnylin is this now matching to any string after the embed param?  I think I'd prefer a narrower thing; if the intent is to support arbitrary future query params, a repeating match on `&([a-zA-Z0-9-]*)=([a-zA-Z0-9-]*)` (or something like that; I'm not sure that's totally correct) seems safer.